dev/pm2.ts

```diff
@@ -0,0 +1,799 @@
+const generators: Record<string, Fig.Generator> = {
+  jsonFileGenerator: {
+    template: "filepaths",
+    filterTemplateSuggestions: function (paths) {
+      const suffix = ".json";
+      return paths.filter((file) => {
+        return file.name.indexOf(suffix, this.length - suffix.length) >= 0;
+      });
```

Comment:
I think we can simplify this a lot:
```suggestion
    filterTemplateSuggestions: function (paths) {
      const suffix = ".json";
      return paths.filter((file) => file.endsWidth(suffix));
```